libbeat/beat/beat.go

```diff
@@ -118,7 +122,7 @@ type BeatConfig struct {
 // Beat (e.g. packetbeat or topbeat). version is version number of the Beater
 // implementation. bt is Beater implementation to run.
 func Run(name, version string, bt Beater) error {
-	return newInstance(name, version, bt).launch(true)
+	return newInstance(name, version, bt).launch()
 }
 
 // instance contains everything related to a single instance of a beat.
```

Comment:
Not really related to this PR, but I'm curious why you created a instance type as Beater is already part of the Beat instance (BT)